Sources/Client/Extensions/Date+Extensions.swift

```diff
@@ -0,0 +1,14 @@
+//
+//  Date+Extensions.swift
+//  StreamChatCore
+//
+//  Created by Alexey Bukhtin on 04/04/2019.
+//  Copyright © 2019 Stream.io Inc. All rights reserved.
+//
+
+import Foundation
+
+extension Date {
+    /// A default date.
+    public static let `default` = Date()
```

Comment:
But why? `Date()` will be different every time the app launches (right? not sure). Why not `Date.timeIntervalSinceReferenceDate`?